string  command
float64 value
time    time
---
bool success
string message
